cppcache/include/geode/ResultCollector.hpp

```diff
@@ -50,8 +50,8 @@ namespace client {
  *  <br>
  *  <pre>
  * auto rc = FunctionService::onRegion(region)
- *                                      ->withArgs(args)
- *                                      ->withFilter(keySet)
+ *                                      .withArgs(args)
+ *                                      .withFilter(keySet)
  *                                      ->withCollector(new
```

Comment:
This line should probably be dot not arrow.

Reply:
Done.